data-source/json/test_schema_theme_census.json

```diff
@@ -0,0 +1,68 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "A questionnaire to check schema census theme.",
+    "theme": "census",
+    "description": "A questionnaire to check schema census theme.",
+    "messages": {
+        "NUMBER_TOO_LARGE": "Number is too large",
+        "NUMBER_TOO_SMALL": "Number cannot be less than zero",
+        "INVALID_NUMBER": "Please enter an integer"
+    },
```

Comment:
These `messages` can be removed to keep the schema as simple as possible.

Reply:
Fixed, thanks!